code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -193,11 +195,13 @@
 	.=..()
 	if(.)
 		embed_mult = 0
+		armor_penetration = ARMOR_PEN_MASSIVE
 		playsound(user, 'sound/weapons/saberon.ogg', 50, 1)
 
 /obj/item/weapon/tool/sword/katana/nano/turn_off(mob/user)
 	..()
 	embed_mult = initial(embed_mult)
+	armor_penetration = initial(armor_penetration)
```

Comment:
Dosnt this reset its AP to base rather then how ever it is with mods? Making it null mods after being on and off again.

Reply:
Yes, but there are no mods that currently change AP afaik

Damage is handled by another Var

Reply:
The only mod that affects ap is strictly for guns at this time